src/api/Jalasoft.TeamUp.Resumes.API/Controllers/PostResume.cs

```diff
@@ -0,0 +1,37 @@
+namespace Jalasoft.TeamUp.Resumes.API.Controllers
+{
+    using System.IO;
+    using System.Net;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+    using Newtonsoft.Json;
+
+    public class PostResume
+    {
+        private readonly IResumesService resumesService;
+
+        public PostResume(IResumesService resumesService)
+        {
+            this.resumesService = resumesService;
+        }
+
+        [FunctionName("resume")]
+        [OpenApiOperation(operationId: "createResume", tags: new[] { "Resumes" })]
+        [OpenApiRequestBody("application/json", typeof(Resume), Description = "JSON request body containing { FirstName, LastName, Email, Phone, Summary}")]
```

Comment:
please double-check the request,  so far I saw in the model you'll be posting more attributes than { FirstName, LastName, Email, Phone, Summary}

Reply:
Added attribute picture

Reply:
Resolved